application_manager/utils/ssh.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2017 UFCG-LSD.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paramiko
+
+
+def get_connection(ip, username=None,
+                       password=None, key_path=None):
```

Comment:
application_manager/utils/ssh.py:20:24: E127 continuation line over-indented for visual indent
                       password=None, key_path=None):